src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -161,7 +163,7 @@ class UnifiedShareModal extends React.Component<Props, State> {
 
         this.state = {
             emailSharedLinkContacts: [],
-            inviteCollabsContacts: [],
+            inviteCollabsContacts: this.props.initiallySelectedContacts || [],
```

Comment:
i think you can just do `props. initiallySelectedContacts` since `props` is an argument to the constructor